packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -570,21 +581,25 @@ export class ConversationService extends TypedEventEmitter<Events> {
           throw new Error('Qualified conversation id is missing in the event');
         }
 
-        const mlsConversation = await this.apiClient.api.conversation.getConversation(conversationId);
-
-        if (!isMLSConversation(mlsConversation)) {
-          throw new Error('Conversation is not an MLS conversation');
-        }
-
-        await this.handleConversationEpochMismatch(mlsConversation, () =>
-          this.emit('MLSConversationRecovered', {conversationId}),
-        );
+        await this.recoverMLSConversationFromEpochMismatch(conversationId);
         return;
```

Comment:
same comment here. return await?

Reply:
Simple await is enough, we don't expect to return anything. Will remove redundant `return` keyword